api/api.go

```diff
@@ -279,19 +268,19 @@ func processCreateDefaults(c libnetwork.NetworkController, nc *networkCreate) {
 		nc.NetworkType = c.Config().Daemon.DefaultDriver
 	}
 	if nc.NetworkType == BridgeNetworkDriver {
```

Comment:
I guess we can remove this special case completely.
I dont see a point in having to deal with BridgeName at this layer